### DISCUSSION TITLE
Colour Changes Upon Export - ".mov" to ".mov"

### DISCUSSION THREAD

**Discussion by JellyBeananator** [mifi/lossless-cut#968] [Support]
So, being overwhelmed by the vast amount of export format options, I decided to export a video to the same format the video was originally recorded in, which is ".mov". I cut the segment, and exported the clip successfully with "Normal Cut" ("Keyframe Cut" exported but showed no video/sound), but then when comparing the exact same segments of the original video and the cut clip, the colour was different. Is there something glaringly obvious that I'm missing? Or is it a common feature I've overlooked considering I'm new to using the software?

Below/attached is a screenshot comparing the two:

- original (top), brighter/more vibrant red
- cut down clip (bottom), a deeper red of the jacket and more of a contrast in the shades of blue in the background.

<img width="1189" alt="Screenshot 2022-01-07 at 13 55 08" src="https://user-images.githubusercontent.com/97295225/148553954-a1e2ecdd-ada8-4ffc-862a-dc66d14d30a0.png">

**Comment by mifi** [comment]
maybe I'm getting old but I cannot really tell any difference. If there is in fact a difference, then that is probably due to some metadata like a color profile that got lost during conversion (makes losslesscut sound ironic huh?). Actually no video data has been lost, it's just the color profile that tells the device's screen how to render the colors that is lost. One thing you could try is exporting with "Preserve all MP4/MOV metadata". Or try mp4/mov/mkv and see if it all makes a difference. If not then it could be a bug in ffmpeg that might be fixed in future versions by the ffmpeg team and will also eventually make it into lossless-cut.

**Reply by ktorenz** [reply]
I have the same problem. The most obvous difference is the highlight. Original MOV video has brighter highlight, while the exported segment has less level of highlight.
